x/stakeibc/migrations/v2/convert.go

```diff
@@ -11,8 +11,6 @@ func convertToNewValidator(oldValidator oldstakeibctypes.Validator) stakeibctype
 	return stakeibctypes.Validator{
 		Name:                 oldValidator.Name,
 		Address:              oldValidator.Address,
```

Comment:
I realize these fields are unused but why are we changing the v2 upgrade logic?

Reply:
I see your PR comment
```
I did test by running the old binary, and then querying the host zones with this binary. The removed fields did not appear and the query did not error - indicating that this new binary can successfully deserialize the old store (which should mean we do not need a state migration).
```
If a new node is catching up and processes an add-validator tx from before this upgrade but after v2, won't it have issues writing the val with the extraneous fields and mismatch with what was written by the current vals (who wrote the extraneous fields to store)?

Reply:
> I realize these fields are unused but why are we changing the v2 upgrade logic?

It's a very good question. The reason I changed these files was to get around a build error - they use the most updated Validator data type which doesn't have the commission status field.

I was a bit worried about breaking the old migration code. And I considered adding the old types here (old as in what's on main, not v2). But decided against it because:
   1) I would have to create a new v3 folder which makes it appear as if there's a migration for this upgrade which there is not, and 
  2) I don't think this code really matters anymore because it's only executed by the v5 upgrade handler which must be run with the v5 binary.

But that said, I'm curious to hear your take!

Reply:
> If a new node is catching up and processes an add-validator tx from before this upgrade but after v2, won't it have issues writing the val with the extraneous fields and mismatch with what was written by the current vals (who wrote the extraneous fields to store)?

I don't think they will because they'll have to be running the v6 or v7 binary which have the old types in there. The only situation where they would have this mismatch is if they were running the v5 upgrade logic from the v8 binary, which I don't think happens

Reply:
makes sense, resolving